knimin/handlers/ag_edit_participant.py

```diff
@@ -13,6 +13,9 @@ def get(self):
         email = self.get_argument('email', None)
         if email is not None:
             login = db.get_login_by_email(email)
+            if not login:
+                raise ValueError("Couldn't retrieve the login information for "
```

Comment:
If it is the standard within labadmin to just raise, then fine for the time being but this should be noted and addressed later